build.sbt

```diff
@@ -37,7 +37,19 @@ def wartRemoverSettings(ignoreFiles: File ⇒ Seq[File] = _ ⇒ Seq.empty[File])
     Wart.Nothing,
     Wart.Overloading,
     Wart.ToString,
-    Wart.Var
+    Wart.Var,
+    Wart.ListAppend,
+    Wart.AsInstanceOf,
+    Wart.PlatformDefault,
+    Wart.NonUnitStatements,
+    Wart.Equals,
+    Wart.JavaSerializable,
+    Wart.StringPlusAny,
+    Wart.SeqApply,
+    Wart.Any,
+    Wart.Product,
+    Wart.Serializable,
+    Wart.GlobalExecutionContext
```

Comment:
I think Var, ListAppend, AsInstanceOf, NonUnitStatements, ,SeqApply and Equals are worth investigating.